tickets_plus/cogs/routines.py

```diff
@@ -91,6 +92,51 @@ async def before_clean_status(self):
         """
         await self._bt.wait_until_ready()
 
+    @tasks.loop(minutes=2, seconds=30)
```

Comment:
Potentially worth making these more configurable if the bot is self-hostable? Eg. Putting variables into config.

Reply:
Hmm. Maybe something like runtimeconfig.json ? For more 'advanced' settings that have reasonable defaults. Wouldn't put this in the normal config.

Reply:
Done.